catalogue/aarch64-GCS/tests/MP+bl+gcsb+dmb.sy-dmb.ld.litmus

```diff
@@ -0,0 +1,23 @@
+AArch64 MP+bl+gcsb+dmb.sy-dmb.ld
```

Comment:
```suggestion
AArch64 MP+bl-gcsb-dmb.sy+dmb.ld
```
I won't comment on all the rest I think you get the point + to move the next thread and - within the thread.